github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
```

Comment:
As I understand, we want to remove `IncludeAdmins` fields. I see that `RequiredPullRequestReviews` contains _only_ that field, and no others, so I understand where your question is coming from.

I would suspect that yes, we want to remove `RequiredPullRequestReviews`, since it has no other fields left, but the GitHub API docs say [some strange/conflicting things](https://developer.github.com/v3/repos/branches/#parameters-1) right now:

> GitHub recommends passing `required_pull_request_reviews` and `enforce_admins`. Passing these fields is not currently required, but may be required soon.

I'm not sure if that's out of date or a typo...